Production/govuk_ios/AppConfig/AppConfigService.swift

```diff
@@ -14,20 +14,28 @@ public final class AppConfigService: AppConfigServiceInterface {
 
     init(configProvider: AppConfigProviderInterface) {
         self.configProvider = configProvider
-        fetchAppConfig()
+        fetchLocalAppConfig()
+        fetchRemoteAppConfig()
     }
 
-    private func fetchAppConfig() {
-        configProvider.fetchAppConfig(
+    private func fetchLocalAppConfig() {
+        configProvider.fetchLocalAppConfig(
```

Comment:
Generally, I think what the plan was with services etc. is to have something internal that is responsible for fetching only remote things, something that is responsible for fetching local things and then the service is responsible for coordinating that.
So serviceClients fetch remote, repositories fetch local (mostly from core data) the service then directs requests to one/ other or both

Reply:
Done